src/meta/user/authorizer.ts

```diff
@@ -156,9 +153,30 @@ const canEditCountryProps = (props: {
 const canEditAssessmentFile = (props: { cycle: Cycle; country: Country; user: User }): boolean =>
   canEditCountryProps({ ...props, allowCollaborator: true })
 
+const canViewCountryFile = (props: {
+  assessment: Assessment
+  cycle: Cycle
+  countryIso: CountryIso
+  user: User
+  assessmentFile: AssessmentFile
+}): boolean => {
+  const { assessment, countryIso, user, cycle, assessmentFile } = props
+  if (Users.isAdministrator(user)) return true
+
+  const userCanView = canView({ assessment, user, countryIso, cycle })
+  const isPublic = assessmentFile?.props.public
+
+  if (isPublic) {
+    return userCanView
+  }
+
+  return userHasRoleForCountryInCycle(user, countryIso, cycle)
```

Comment:
If file is private:

- Admin can see always (line 164)
- User can only see if he/she has a role in cycle/country